deps/nghttp2/lib/includes/nghttp2/nghttp2ver.h

```diff
@@ -29,14 +29,14 @@
  * @macro
  * Version number of the nghttp2 library release
  */
-#define NGHTTP2_VERSION "1.39.1"
+#define NGHTTP2_VERSION "1.33.0"
```

Comment:
is this 1.39.2 or 1.33.0?

Reply:
@devsnek Thanks for noticing, I forgot to run `autoconf` on my checkout of nghttp2. Fixed :)